### ISSUE TITLE
docker-compose service definitions fail 

### ISSUE THREAD

**Issue by yancyribbens** [lightningnetwork/lnd#2343]
### Background

docker/lnd/Dockerfile has outdated dependencies

### Your environment

the issue occurs on both OSX and Ubuntu

### Steps to reproduce

docker-compose run --name alice lnd_btc

### Expected behaviour

should launch lnd_btc service located in docker-compose.yml file

### Actual behaviour

fails with the following:

```
ERROR: Service 'lnd' failed to build: The command '/bin/sh -c apk add --no-cache     git     make &&  cd /go/src/github.com/lightningnetwork/lnd &&  make &&  make install' returned a non-zero code: 2
```

There are currently 3 open PRs to address this issue:
https://github.com/lightningnetwork/lnd/pull/2259
https://github.com/lightningnetwork/lnd/pull/2333
https://github.com/lightningnetwork/lnd/pull/2334

**Comment by halseth** [comment]
#2333 seems simplest. Would merging that one fix the issue?

**Comment by yancyribbens** [comment]
@halseth I disagree #2333 is simpler.  I left comments on the PR, although it would probably also fix the issue.

**Comment by Roasbeef** [comment]
Fixed by #2259.
